tests/function/test_config.py

```diff
@@ -614,3 +615,272 @@ def test_interval_in_virtwho_sysocnfig(self, virtwho, function_sysconfig):
+        globalconf.update('global', 'configs', configs_data['wrong_configs'])
+        result = virtwho.run_service()
+        assert (result['error'] is not 0
+                and result['send'] == 0
+                and result['thread'] == 0
+                and (error in result['log'] for error in configs_data['error']))
+
+    @pytest.mark.tier2
+    @pytest.mark.notLocal
+    def test_owner_in_virtwho_conf(self, virtwho, globalconf, function_hypervisor, hypervisor_data,
+                                   owner_data):
+        """Test the owner option in /etc/virtwho.conf
+
+        :title: virt-who: config: test owner negative option
+        :id: 2b6f68bb-ea7e-4c01-abe5-7c27da3b3d3f
+        :caseimportance: High
+        :tags: tier2
+        :customerscenario: false
+        :upstream: no
+        :steps:
+
+            1. Disable owner option in /etc/virt-who.d/,
+            2. Run virt-who with null owner setting in [defaults] section
+            in /etc/virt-who.conf
+
+        :expectedresults:
+
+            1. Virt-who runs failed with the incorrect owner setting
+        """
+        globalconf.update('global', 'debug', 'True')
+
+        function_hypervisor.delete('owner')
+        globalconf.update('defaults', 'owner', '')
+        result = virtwho.run_service()
+        assert (result['error'] is not 0
+                and result['send'] == 0
+                and result['thread'] == 1
+                and any(error in result['error_msg'] for error in owner_data['null_error']))
+
+    @pytest.mark.tier2
+    @pytest.mark.notLocal
+    def test_hypervisor_id_in_virtwho_conf(self, virtwho, globalconf, function_hypervisor, hypervisor_data,
+                                           register_data, rhsm, satellite):
+        """Test the hypervisor_id negative option in /etc/virtwho.conf
+
+        :title: virt-who: config: test hypervisor_id option
+        :id: fc2dcc791-0182-4309-a3d8-fc899d3938fa
+        :caseimportance: High
+        :tags: tier2
+        :customerscenario: false
+        :upstream: no
+        :steps:
+
+            1. Config the hypervisor_id=hostname in /etc/virt-who.d/virt-who.conf(default)
+            2. Config the hypervisor_id=uuid in /etc/virt-who.conf
+            3. Run virt-who service and check the mapping info
+            4. Delete the host from the register platform
+
+        :expectedresults:
+
+            3. hypervisor_id in /etc/virt-who.d/virt-who.conf has high priority, can find the
+            hypervisor_id is the hostname in the mapping info.
+        """
+        globalconf.update('global', 'debug', 'True')
+        globalconf.update('defaults', 'hypervisor_id', 'uuid')
+
+        result = virtwho.run_service()
+        assert (result['error'] == 0
+                and result['send'] == 1
+                and result['thread'] == 1
+                and result['hypervisor_id'] == hypervisor_data['hypervisor_hostname'])
+        if REGISTER == 'rhsm':
```

Comment:
it seems no need to delete the hosts info from register server side.

Reply:
also move this from old cases, that would be great :)